compiler-rt/lib/builtins/fp_lib.h

```diff
@@ -374,10 +376,10 @@ static __inline fp_t __compiler_rt_fmax(fp_t x, fp_t y) {
 #endif
 }
 
-#elif defined(QUAD_PRECISION) && defined(CRT_HAS_TF_MODE)
+#elif defined(QUAD_PRECISION)
```

Comment:
Looks like this was actually required. Probably needs to be changed to `#elif defined(QUAD_PRECISION) && defined(CRT_HAS_F128)`. @perry-ca Could you open a new PR with this change? See https://github.com/llvm/llvm-project/issues/77898 and https://github.com/llvm/llvm-project/issues/77880

Reply:
I've put up #77981 for review.